src/components/DotIndicatorMessage.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import _ from 'underscore';
+import PropTypes from 'prop-types';
+import {View} from 'react-native';
+import styles from '../styles/styles';
+import Icon from './Icon';
+import * as Expensicons from './Icon/Expensicons';
+import colors from '../styles/colors';
+import variables from '../styles/variables';
+import Text from './Text';
+
+const propTypes = {
+    /** The messages to display  */
+    // eslint-disable-next-line react/forbid-prop-types
+    messages: PropTypes.object,
+
+    /** The type of message, 'error' shows a red dot, 'success' shows a green dot */
+    type: PropTypes.oneOf(['error', 'success']).isRequired,
+
+    /** Additional styles to apply to the container */
+    // eslint-disable-next-line react/forbid-prop-types
+    style: PropTypes.arrayOf(PropTypes.object),
+};
+
+const defaultProps = {
+    messages: null,
+    style: [],
+
+};
+
+const DotIndicatorMessage = props => (
+    <View style={[styles.dotIndicatorMessage, ...props.style]}>
+        <View style={styles.offlineFeedback.errorDot}>
+            <Icon src={Expensicons.DotIndicator} fill={props.type === 'error' ? colors.red : colors.green} height={variables.iconSizeSmall} width={variables.iconSizeSmall} />
+        </View>
+        <View style={styles.offlineFeedback.textContainer}>
+            {_.map(props.messages, (message, i) => (
+                <Text key={i} style={styles.offlineFeedback.text}>{message}</Text>
+            ))}
+        </View>
```

Comment:
do we care to wrap this in an `if` block and check if `!_.isEmpty(messages)` so that we don't render anything if there are no messages to show?

Reply:
great idea! opted to go for an early return and flipped the condition you provided since looks like that's what we're doing in most other places